packages/core/src/payment/strategies/braintree/braintree-hosted-form.ts

```diff
@@ -254,6 +255,26 @@ export default class BraintreeHostedForm {
         }
     }
 
+    private _mapErrors(fields: BraintreeHostedFieldsState['fields']): BraintreeFormErrorsData {
+        const errors: BraintreeFormErrorsData = {};
+
+        if (fields) {
+            for (const [key, value] of Object.entries(fields)) {
+                if (value) {
+                    const { isValid, isEmpty, isPotentiallyValid } = value;
+
+                    errors[key as keyof BraintreeFormErrorsData] = {
```

Comment:
I tried to eliminate this `as` keyword but can't think of a neat way to do so. The way I can think of is either add [key: string] accessor to the BraintreeFormErrorsData or manual mapping the content with if/else, would like to see what will the advice here.